PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,28 @@
+<!-- Thank you for your contribution!
+
+     Before submit your Pull Request, make sure you picked
+     the right branch:
+
+     - If you propose a new feature or improvement, select "master"
+       as a target branch;
+     - If this is code bug fix or documentation correction, select
+       the lastest release branch (which looks like "0.xx") -->
+
+## What these changes does?
+
+<!-- Please give a short brief about these changes. -->
+
+## How to test your changes?
+
+<!-- Describe how we can test your changes if they provides
+     any notable behaviour for the end users. -->
+
+## Related issue number
+
+<!-- Is there any issue opened that get fixed by this? -->
+
+## Checklist
+
+- [ ] Code is written and well
+- [ ] Tests for the changes are provided
```

Comment:
I mean this by `Code is written and *well*`. And flake check is a part of `make test` which is recommended in CONTRIBUTING as the way to test your changes. Moreover, TravisCI runs flake checks. So, if you check these two point with flake issues then this only means that you didn't test anything or made it in own way that may be not a part of common workflow.